src/api_handler.js

```diff
@@ -0,0 +1,60 @@
+/*
+  eslint-disable no-unused-vars, no-alert, no-mixed-operators, import/prefer-default-export,
+  no-empty
+*/
```

Comment:
are all these warnings actually thrown by eslint or you just copied these into most of your files. For examples, I do not see why  "import/prefer-defautt-export" would come up when you are exporting multiple modules and not importing anything at all  :thinking: